users.go

```diff
@@ -75,6 +75,7 @@ type User struct {
 	WebsiteURL                     string             `json:"website_url"`
 	Organization                   string             `json:"organization"`
 	JobTitle                       string             `json:"job_title"`
+	Bot                            bool               `json:"bot"`
```

Comment:
Please place the new field between `Bio` and `Location` following the order of the docs...

Reply:
Sure.